src/tools/generate/generate_sitemap.js

```diff
@@ -91,14 +91,21 @@ const set_min_date = (date) => {
   }
 };
 
-const strip_html_extension = (url) => {
+const convert_file_name = (url) => {
   if ( url.substr(url.length - 10) == "index.html" ) {
     return url.substr(0, url.length - 10);
   };
   if ( url.substr(url.length - 5) == ".html" ) {
-    return url.substr(0, url.length - 5);
+    if ( url.substr(url.length - 28) == "accessibility_statement.html" ) {
```

Comment:
Does this simplification still work?
```suggestion
    if ( url.endsWith("accessibility_statement.html") ) {
```

Reply:
Yes it does. Clever JavaScript! 😀